packages/flutter_tools/lib/src/build_system/targets.dart

```diff
@@ -0,0 +1,329 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../asset.dart';
+import '../base/file_system.dart';
+import '../base/io.dart';
+import '../base/process_manager.dart';
+import '../build_info.dart';
+import '../compile.dart';
+import '../devfs.dart';
+import '../project.dart';
+
+import 'build_system.dart';
+
+/// Build definitions.
```

Comment:
There are a bunch of different targets with different purposes all lumped together in this file. It might be helpful to split them apart under like, build_system/targets/ or build_system/builtin_targets/